auth/v1.js

```diff
@@ -688,6 +698,41 @@ api.declare({
   return res.reply();
 });
 
+/** Expand a scopeset */
+api.declare({
+  method:     'delete',
```

Comment:
he he, interesting choice of method... May I suggest `get`.. 

You have same copy/pasta issue below :)